### PR TITLE
[#305] add home and pregame mobile styles

### PR DESCRIPTION
**Pull request by benshumaker**
Closes: #305 
Closes: #306

### CODE REVIEW

File: frontend/packages/web/src/components/features/CreateRoomForm/CreateRoomForm.tsx [hunk 1]
```diff
@@ -72,17 +78,17 @@ export function CreateRoomForm() {
           onKeyDown={handleKeyDown}
         ></TextInput>
 
-        <WhiteBackgroundText>Gamemode:</WhiteBackgroundText>
-        <Dropdown name="roomForm_game_mode" onChange={(e) => handleOnchange(e)}>
+        <NonMobileWhiteBackgroundText>Gamemode:</NonMobileWhiteBackgroundText>
```
Comment (izakfr) [inline frontend/packages/web/src/components/features/CreateRoomForm/CreateRoomForm.tsx:81]: Is there a better way to do / name this. `NonMobileWhiteBackgroundText` doesn't look right to me.

File: frontend/packages/web/src/components/pages/Homepage/Homepage.styled.tsx [hunk 2]
```diff
@@ -1,13 +1,22 @@
 import React from 'react';
 import styled from 'styled-components';
 
-export const FlexRow = styled.div`
+export const HomepageTopContainer = styled.div`
```
Comment (izakfr) [inline frontend/packages/web/src/components/pages/Homepage/Homepage.styled.tsx:4]: Can you name these better? Something like HomepageOuterContainer and HomepageInnerContainer?

Comment (izakfr) [reply frontend/packages/web/src/components/pages/Homepage/Homepage.styled.tsx:4]: Anything like ...V2 or Second... is a trigger to me lol

**Review by izakfr — CHANGES_REQUESTED** [review]

**Comment by benshumaker** [comment]
I am going to revise this PR so you don't have to scroll to see the full game mode preview.

File: frontend/packages/web/src/components/features/CreateRoomForm/CreateRoomForm.tsx (hunk 1 above)
Comment (benshumaker) [reply frontend/packages/web/src/components/features/CreateRoomForm/CreateRoomForm.tsx:81]: I'll find a cleaner way to do this later. Soon I'll be doing a component/style refactor. I thought of a cleaner prop-based way to do this.

**Review by izakfr — APPROVED** [review]
I have very closely analyzed the contents of this pull request.

**Pull request merged**